lib/worldwide/regions_loader.rb

```diff
@@ -102,6 +102,7 @@ def apply_territory_attributes(region, spec)
       region.currency = Worldwide.currency(code: currency_code) unless currency_code.nil?
       region.flag = spec["emoji"]
       region.format = spec["format"]
+      region.format_extended = spec["format_extended"] || {}
```

Comment:
What are your thoughts on defaulting to `spec["format"]` instead of an empty hash? I imagine that a client that supports additional fields would always request the extended format, regardless of whether the country actually has additional fields. This way they won't need to implement a fallback when there is no extended format.